lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -2294,8 +2300,16 @@ namespace Js
             case UDAT_TIMEZONE_SPECIAL_FIELD:
             case UDAT_TIMEZONE_LOCALIZED_GMT_OFFSET_FIELD:
             case UDAT_TIMEZONE_ISO_FIELD:
+            case UDAT_TIMEZONE_ISO_LOCAL_FIELD:
                 typeString = _u("timeZoneName"); break;
+#if defined(ICU_VERSION) && ICU_VERSION == 55
```

Comment:
@jackhorton Is any other version of ICU unaffected by this?

Reply:
Correct